arbor/backends/gpu/shared_state.hpp

```diff
@@ -27,8 +27,11 @@ namespace gpu {
  *     Xi_     cai              internal calcium concentration
  *     Xo_     cao              external calcium concentration
  */
+ struct ARB_ARBOR_API ion_state {
+    bool write_eX;          // is eX written?
+    bool write_Xo;          // is Xo written?
+    bool write_Xi;          // is Xi written?
```

Comment:
Rename to have tralining underscore, e.g. `write_eX_`, to be in line with naming scheme of other member variables.